pkgs/by-name/kr/krename/package.nix

```
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  fetchurl,
```

We do not use this currently
```suggestion
  # fetchurl,
```